README.md

```diff
@@ -49,8 +51,13 @@ They are:
 
 **dvsampler** A script that downloads or creates DV files to use as samples. Requires FFmpeg.
 
+**dvmerge** A script that takes multiple transfers of the same tape containing errors and combines them to create one file with the best information available for each problematic frame. 
```

Comment:
The merge function is actually part of 'dvrescue'. See `dvrescue -h` on a recent build.

Reply:
Okay, gotcha. See new edits and let me know if that wasn't what you meant or if there were additional links that I missed.